src/test/java/frc/robot/filters/TestTurnToggle.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.filters;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class TestTurnToggle {
+    private static final double SMALLDIFFERENCE = 0.01;
+    
+    @Test
+    public void testTurnToggleOff() {
+        TurnToggleFilter filter = new TurnToggleFilter();
+        DriveInput DI = new DriveInput (0.5, 0.75, 0.6);
```

Comment:
these numbers are duplicated below. I presume they should match-- if so, use a constant to make it clear they are supposed to